breeze

```diff
@@ -757,13 +775,15 @@ fi
 # Those files are mounted into container when run locally
 # .bash_history is preserved and you can modify .bash_aliases and .inputrc
 # according to your liking
+if [[ -d "${MY_DIR}/.bash_history" ]]; then
```

Comment:
Thanks for that. This will probably fix my bash history. 😸

Reply:
YEAH.